lib/src/showcase.dart

```diff
@@ -252,13 +254,25 @@ class Showcase extends StatefulWidget {
   /// Defaults to 7.
   final double toolTipSlideEndDistance;
 
+  /// Title alignment within tooltip widget
```

Comment:
Can you please add widget word in there?
```suggestion
  /// Title widget alignment within tooltip widget
```